Samples/GitHubLabeler/GitHubLabeler/GitHubLabeler.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
```

Comment:
Same comment here as above -

Move `<LangVersion>latest</LangVersion>` into the unconditional property group, and remove these two.